src/generate/utils/execute/gadget/OperationExecutor_Jetpack.cpp

```diff
@@ -9,6 +9,9 @@ OperationExecutor::executeJetpack(const spy::gameplay::State_AI &state, const sp
     spy::gameplay::State_AI s = state;
 
     auto character = s.getCharacters().getByUUID(op.getCharacterId());
+    if (spy::gameplay::Movement::getMoveDistance(character->getCoordinates().value(), op.getTarget()) <= 1) {
```

Comment:
Muss man das hier nicht mit den MP vergleichen?

Reply:
Du meinst, dass jetpack Nutzung für eine Distanz von einem Feld ok ist, wenn man keine mp hat?

Reply:
Auch. Meinte aber auf ein Feld in Distanz <= MP

Reply:
Statt halt in der eval